ansible/roles/git_lfs/README.md

```diff
@@ -9,5 +9,7 @@ None.
 ## Manual Installation
 
 ```bash
+curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | sudo bash
```

Comment:
I think you don't need this at all, it is in here: https://packages.ubuntu.com/jammy/git-lfs

Reply:
Also in the role too.

Reply:
Okay, I'll try it.